network/test/meshengine_test.go

```diff
@@ -220,7 +239,11 @@ func (suite *MeshEngineTestSuite) targetValidatorScenario(send ConduitSendWrappe
 	}
 
 	// allow nodes to heartbeat and discover each other
-	time.Sleep(5 * time.Second)
+	// each node will register ~D protect messages, where D is the default out-degree (6)
+	// see https://github.com/libp2p/go-libp2p-pubsub/blob/0c7092d1f50091ae88407ba93103ac5868da3d0a/gossipsub.go#L33
+	for i := 0; i < 6*count; i++ {
```

Comment:
```suggestion
	for i := 0; i < pubsub.GossipSubD  *count; i++ {
```
then can't we do this?

Reply:
Doh.